src/components/layout/layout.js

```diff
@@ -4,7 +4,7 @@ import PropTypes from 'prop-types'
 import styled from 'styled-components'
 import useGTMData from '../hooks/use-gtm-data'
 import Copyright from './copyright'
-import { Nav, NavStatic, NavPartners, NavInterim } from './nav'
+import { Nav, NavStatic, NavPartners, NavInterim, NavSecurity } from './nav'
```

Comment:
Can we please name this TransparentNav or something along those lines? AFAIK, homepage is using this style of a header as well. This is to avoid many switch case statements for the Nav types when we transition the other pages later.

Reply:
Guess you're referring to Nav from about-us branch here...that has a transparent nav.